interoperability/git.md

```diff
@@ -39,6 +53,12 @@ This will update everything:
 brew update && brew upgrade `brew outdated`
 ```
 
+### Windows
+
+Beside the Windows installation on the main download page there is also
```

Comment:
Hello, yes the main download link is mentioned in the first paragraph before the operating systems.